packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/zerto/stateBadge/translations/Messages_fr_FR.json

```
@@ -0,0 +1,11 @@
+{
+  "dedicatedCloud_zerto_site_status_vpn_creating": "Configuration en cours…",
```

In french, there is a space before punctuation
```suggestion
  "dedicatedCloud_zerto_site_status_vpn_creating": "Configuration en cours …",
```